src/InfoPanel/InfoPanel.css

```diff
@@ -0,0 +1,36 @@
+.InfoPanel {
```

Comment:
Should we be committing this css file? We already have the less file and no other components in the `src` directory seem to include css files.